### DISCUSSION TITLE
useControl vs useInputControl

### DISCUSSION THREAD

**Discussion by ensidialabs** [edmundhung/conform#680] [General]
Hello,
I would like to know in which cases it is better to use unstable_useControl and in which useInputControl?
The examples indicate unstable_useControl, and the guide on the website useInputControl

**Comment by lilouartz** [comment]
@edmundhung Would greatly appreciate your response here.

**Comment by marcomuser** [comment]
I've been using useControl exclusively as useInputControl gave me sometimes issues (mostly in more dynamic form sections) whereas useControl didn't. I guess the docs have not yet been updated because useControl was not marked stable yet.
